src/PSGitCompletions/GitRemote.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PowerCode
+{
+    public struct GitRemote
```

Comment:
Should probably be a class or a record. One way or another, we should be able to check identity and such in an easy way.

Reply:
Added ability to check identity based on the commit